tools/macos_buildenv.sh

```diff
@@ -22,19 +22,20 @@ THIS_SCRIPT_NAME=${BASH_SOURCE[0]}
 if [ -n "${BUILDENV_ARM64}" ]; then
     VCPKG_TARGET_TRIPLET="arm64-osx-min1100-release"
     BUILDENV_BRANCH="2.5-rel"
-    BUILDENV_NAME="mixxx-deps-2.5-$VCPKG_TARGET_TRIPLET-bea61fb"
-    BUILDENV_SHA256="0c99cf740f16fd3a1b81ad17c5dcc94e9d18c6cdc92c3f6ca058fe0297a7cddd"
+    BUILDENV_NAME="mixxx-deps-2.5-$VCPKG_TARGET_TRIPLET-a28da9d"
+    BUILDENV_SHA256="fb3a85bac430c01effe68a6c53a58105aa926975161f1412739f3985ea3c0097"
 else
     if [ -n "${BUILDENV_RELEASE}" ]; then
         VCPKG_TARGET_TRIPLET="x64-osx-min1015-release"
         BUILDENV_BRANCH="2.5-rel"
-        BUILDENV_NAME="mixxx-deps-2.5-$VCPKG_TARGET_TRIPLET-bea61fb"
-        BUILDENV_SHA256="48c89e22e3c82b299d2e216a7a73c52a7b1c6961a2cce883df120ce610c09403"
+        BUILDENV_NAME="mixxx-deps-2.5-$VCPKG_TARGET_TRIPLET-a28da9d"
+        BUILDENV_SHA256="345945e377993b50d3b45448c89a1b63c571b58bdf5cbefe1ac455be45c53d6f"
     else
         VCPKG_TARGET_TRIPLET="x64-osx-min1015"
         BUILDENV_BRANCH="2.5"
-        BUILDENV_NAME="mixxx-deps-2.5-$VCPKG_TARGET_TRIPLET-5d58718"
-        BUILDENV_SHA256="c0b6d25c9bb5f7951014a06a03371e4190e0f8977805bf56c09dfa1ef2b285ae"
+        BUILDENV_NAME="mixxx-deps-2.5-$VCPKG_TARGET_TRIPLET-03df0a6"
+        mixxx-deps-2.5-x64-osx-min1015-03df0a6
```

Comment:
```suggestion
```

Reply:
Upsi, I have rebased this extra line aways